README.md

```diff
@@ -336,27 +102,12 @@ The `.subscribe` method can take [a few different forms](https://github.com/Reac
    npm run lint:fix
    ```
 
-### Developing with the Example App
-The LLM example app can be a quick way to test out changes to the LLM plugin.
-
-To use the example app in conjunction with the LLM plugin:
-1. Clone the [llm example app](https://github.com/grafana/grafana-llmexamples-app/tree/main)
-2. Update the following fields in `docker-compose.yaml` in the llm example app
-- comment out # GF_INSTALL_PLUGINS: grafana-llm-app
-- Add the following volume:
-```
-<some-parent-path>/grafana-llm-app/dist:/var/lib/grafana/plugins/grafana-llm-app
-```
-3. Follow the instructions in the llm example app to run the app
-
 ## Release process
```

Comment:
Do we need a release process the `@grafana/llm`?

Reply:
We do; I want to probably add that separately, I'll open an issue for it.